app/views/shared/check_answers/_opponent_details.html.erb

```diff
@@ -1,150 +1,136 @@
+      <% unless read_only %>
+        <% card.with_action do %>
+          <%= govuk_link_to(t("generic.change"),
+                            providers_legal_aid_application_opponents_mental_capacity_path(@legal_aid_application), visually_hidden_text: t(".mental_capacity.heading")) %>
+        <% end %>
       <% end %>
-      <% unless parties_mental_capacity.understands_terms_of_court_order? %>
-        <%= summary_list.with_row do |row| %>
-          <%= row.with_key(text: t(".mental_capacity.reasons"), classes: "govuk-!-width-one-half") %>
-          <%= row.with_value(text: parties_mental_capacity.understands_terms_of_court_order_details) %>
-          <%= row.with_action(
-                text: t("generic.change"),
-                href: providers_legal_aid_application_opponents_mental_capacity_path(@legal_aid_application),
-                visually_hidden_text: t(".mental_capacity.reasons"),
-              ) %>
+      <%= card.with_summary_list(actions: false) do |summary_list| %>
+        <%= summary_list.with_row(html_attributes: { id: "app-check-your-answers__opponent_understands_terms_of_court_order" }) do |row| %>
+          <%= row.with_key(text: t(".mental_capacity.understands_terms"), classes: "govuk-!-width-one-half") %>
+          <%= row.with_value(text: yes_no(parties_mental_capacity.understands_terms_of_court_order)) %>
+        <% end %>
+        <% unless parties_mental_capacity.understands_terms_of_court_order? %>
+          <%= summary_list.with_row do |row| %>
+            <%= row.with_key(text: t(".mental_capacity.reasons"), classes: "govuk-!-width-one-half") %>
+            <%= row.with_value(text: parties_mental_capacity.understands_terms_of_court_order_details) %>
+          <% end %>
         <% end %>
       <% end %>
     <% end %>
   <% end %>
 
   <% if task_list_includes?(@legal_aid_application, :domestic_abuse_summary) %>
-    <%= govuk_summary_list(card: { title: t(".domestic_abuse.heading") }, actions: !read_only) do |summary_list| %>
-      <% unless domestic_abuse_summary.warning_letter_sent? %>
-        <%= summary_list.with_row(
-              html_attributes: { id: "app-check-your-answers__opponent_warning_letter_sent" },
-            ) do |row| %>
-          <%= row.with_key(text: t(".domestic_abuse.warning_letter_reasons"), classes: "govuk-!-width-one-half") %>
-          <%= row.with_value(text: domestic_abuse_summary.warning_letter_sent_details) %>
-          <%= row.with_action(
-                text: t("generic.change"),
-                href: providers_legal_aid_application_domestic_abuse_summary_path(@legal_aid_application),
-                visually_hidden_text: t(".domestic_abuse.warning_letter_reasons"),
-              ) %>
+    <%= govuk_summary_card(title: t(".domestic_abuse.heading"), html_attributes: { id: "app-check-your-answers__domestic_abuse" }) do |card| %>
+      <% unless read_only %>
+        <% card.with_action(actions: false) do %>
```

Comment:
I think this `(actions: false)` has been put in the wrong place by accident, and should be on line 53

Reply:
👍 thanks good spot!